### ISSUE TITLE
Install script adds duplicate keys

### ISSUE THREAD

**Issue by asymmetric** [dapphub/dapp-tools#3]
My `nix.conf` already contained a `substituers` key, and the `install` script added another one at the bottom of the script.

We should instead append to the existing key.

**Comment by nanexcool** [comment]
Yes, great idea! We should discuss this. What we add in `nix.conf` is basically what you get by installing `cachix` then running `cachix use dapp`. But... I noticed on Macs sometimes `cachix` installs from source and compiles and takes a long time.

This is why we only modify the `nix.conf` file naively. Since it's 3 lines (with 1 comment) we could check for the existance of these lines and only add if not found.
